docs/api/blueprint/user/users.apib

```diff
@@ -559,9 +559,9 @@ Authorized user should be same as user in request body or must be admin.
                   "twitter-url": "http://twitter.com/twitter",
                   "instagram-url": "http://instagram.com/instagram",
                   "google-plus-url": "http://plus.google.com/plus.google",
-                  "thumbnail-image-url": "http://example.com/example.png",
-                  "small-image-url": "http://example.com/example.png",
-                  "icon-image-url": "http://example.com/example.png"
+                  "thumbnail-image-url": "https://image.flaticon.com/icons/png/64/126/126486.png",
+                  "small-image-url": "https://image.flaticon.com/icons/png/64/126/126486.png",
+                  "icon-image-url": "https://image.flaticon.com/icons/png/64/126/126486.png"
```

Comment:
These are read only and should be removed

Reply:
remove these fields or revert the change? Removing from the documentation response will be like the documentation is incomplete? Your call, tell me what

Reply:
No, they are discarded. So, the documentation will be complete

Reply:
@iamareebjamal

Reply:
@iamareebjamal Test are failing because the request becomes incomplete

Reply:
No, they are failing because it's invalid JSON

Reply:
reverting this change to the previous one.

Reply:
You can't have trailing commas in JSON